ppl/src/main/java/org/opensearch/sql/ppl/parser/AstExpressionBuilder.java

```diff
@@ -332,8 +335,11 @@ private List<UnresolvedExpression> relevanceArguments(RelevanceExpressionContext
     // all the arguments are defaulted to string values
     // to skip environment resolving and function signature resolving
     ImmutableList.Builder<UnresolvedExpression> builder = ImmutableList.builder();
+    ParserRuleContext field = ctx.field;
+    if (field == null)
+      field = ctx.fields;
     builder.add(new UnresolvedArgument("field",
-        new Literal(StringUtils.unquoteText(ctx.field.getText()), DataType.STRING)));
+        new Literal(StringUtils.unquoteText(field.getText()), DataType.STRING)));
     builder.add(new UnresolvedArgument("query",
         new Literal(StringUtils.unquoteText(ctx.query.getText()), DataType.STRING)));
     ctx.relevanceArg().forEach(v -> builder.add(new UnresolvedArgument(
```

Comment:
1. Should we create 2 different functions as `relevanceArguments`?
2. Should we skip `StringUtils.unquoteText` for `fields`?